### DISCUSSION TITLE
Sandbox console maintence tasks - Monday 25 November 2024 from 15:30 CET

### DISCUSSION THREAD

**Discussion by jescriu** [INSPIRE-MIF/helpdesk-geoportal#232] [Announcements]
Dear INSPIRE Geoportal users,

The access to the [sandbox console](https://inspire-geoportal.ec.europa.eu/sandboxconsole) will be disabled today from 15:30 CET to perform maintenance tasks for the deployment of an updated version of the Link-Checker. The update incorporates the correction some bugs detected in the processing of the protocol field (from the onlineResource), while relaxing the requirements for matching INSPIRE identifiers between the services and the dataset/series metadata files.

Once the Link-Checker is duly updated, we will restore the sandbox service and update this notification. The intervention is expected to last approximately 2 hours.

**Comment by jescriu** [comment]
The access to the sandbox console has been successfully restored. The sandbox has been duly updated with the new version of the Link-Checker referred above.

**Comment by dbu-spw** [comment]
Dear @jescriu,

The sandbox console seems to be down this morning with a "Service Unavailable" message. 
Is-it on the sandbox side or my network ?

Thkx for your help
